nautobot/core/testing/views.py

```diff
@@ -459,6 +461,7 @@ def test_edit_object_with_permission(self):
                 objectchanges = lookup.get_changes_for_model(instance)
                 self.assertEqual(len(objectchanges), 1)
                 self.assertEqual(objectchanges[0].action, extras_choices.ObjectChangeActionChoices.ACTION_UPDATE)
+                self.assertEqual(instance.last_updated_by, self.user)
```

Comment:
Might need/want to use a different user account specifically for this test - since all of our generic tests pretty much use the same single user account, this isn't a very meaningful test as currently implemented.